BE/tools/final_answer.py

```
@@ -0,0 +1,14 @@
+from typing import Any, Optional
```

Ruff (0.8.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)